fml/task_queue_id.h

```diff
@@ -0,0 +1,26 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_FML_TASK_QUEUE_ID_H_
+#define FLUTTER_FML_TASK_QUEUE_ID_H_
+
+#include "flutter/fml/logging.h"
+
+namespace fml {
+
+class TaskQueueId {
+ public:
+  static const size_t kUnmerged;
+
+  explicit TaskQueueId(size_t value) : value_(value) {}
+
+  operator int() const { return value_; }
```

Comment:
Hah, I like this as a way of making sure IDs are initialized to defaults. But perhaps the cast should not also perform a conversation. So `operator size_t() const;` perhaps?

Reply:
Done